services/headless-lms/server/src/programs/seed/seed_courses.rs

```diff
@@ -2204,6 +2204,277 @@ pub async fn seed_switching_course_instances_course(
     Ok(course.id)
 }
 
+pub async fn seed_chatbot_course(
```

Comment:
Split the seed courses into multiple files. Rename the existing file to seed_courses/mod.rs,  and then add a new submodule for this function only seed_chatbot_course.